Cargo.toml

```diff
@@ -6,20 +6,23 @@ members = [
 ]
 
 [workspace.dependencies]
-anchor-client = "0.27.0"
-anchor-lang = "0.27.0"
-anchor-spl = "0.27.0"
-fixed = { path = "./3rdparty/fixed", version = "1.11.0" }
-pyth-sdk-solana = "0.7.0"
+# rev 2af9cc669 is successor of 0.28.0
+anchor-client = { git = "https://github.com/coral-xyz/anchor.git", rev = "2af9cc669", version = "0.28.0" }
```

Comment:
should use 0.28.0 instead

Reply:
Really? Wasn't there a good reason to go to 2af9cc669, due to solana version issues with 0.28.0?

Reply:
Anchor `0.28.0` should work just fine here since Solana crates are also being updated. https://github.com/coral-xyz/anchor/pull/2572 was only for `solana-client 1.14`.

Reply:
thx will do

Reply:
tried with vanilla anchor 0.28 but that gives me this error:

> 39 |     pub openbook_v2_market_external: AccountLoader<'info, Market>,
>    |                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `ZeroCopy` is not implemented for `openbook_v2::state::Market`

you can try by comparing e9b23d587 and fa6e24c90

Reply:
openbook_v2::market has been defined as zero copy. Is the branch latest branch ?